roles/key_inject/tasks/inject.yml

```
@@ -0,0 +1,55 @@
+- name: Inject keys
+  block:
+    - name: Inject | Setting {{ item.type }} pub keys
+      ansible.builtin.set_fact:
+        key_inject_pub_key: "{{ (item.priv_key |
+            infrastructure.chain_operations.subkey_inspect(scheme=(item.scheme | default('sr25519')) )).publicKey }}"
```

Plugins is stored [here](https://github.com/paritytech/ansible-galaxy/tree/main/plugins/filter)
```suggestion
           parity.chain.subkey_inspect(scheme=(item.scheme | default('sr25519')) )).publicKey }}"
```